wp-content/themes/haruoshi_origin/src/scss/archive-works.scss

```
@@ -41,3 +41,70 @@
     object-fit: cover; //IE11では使えないよ
   }
 }
+
+.pager {
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
+}
+
+.pagination {
+  display: flex;
+}
+
+.page-numbers {
+  font-size: 20px;
+  margin-right: 14px;
+  width: 20px;
+  display: inline-block;
+}
+
+.arrow::before {
+  content: "";
+  width: 14px;
+  height: 14px;
+  border-top: solid 2px $main-color;
+  border-right: solid 2px $main-color;
+  position: absolute;
+}
+
+.arrow {
+  &-left::before {
+    transform: rotate(-135deg);
+  }
+  &-right::before {
+    transform: rotate(45deg);
+  }
+}
```

ここ疑似要素じゃなくて普通に `.arrow` に対するスタイルでよかった。
こういう感じ。
```suggestion
.arrow {
  width: 14px;
  height: 14px;
  border-top: solid 2px $main-color;
  border-right: solid 2px $main-color;
  position: absolute;
  &--left {
     transform: rotate(-135deg);
  }
  &--right{
    transform: rotate(45deg);
  }
}
```